### ISSUE TITLE
Aug. 30th commit broke importing project into eclipse

### ISSUE THREAD

**Issue by hoyt85** [google/closure-compiler#2028]
Hey guys,

[This commit](https://github.com/google/closure-compiler/commit/24a4540ba13ca930397265e3f4dd4a42e3372de1) broke the ability to import the project into Eclipse through the method described in the main README.md. Specifically, Eclipse complains that the file at src/com/google/debugging/sourcemap/super/com/google/debugging/sourcemap/SourceMapObjectParser.java declared package does not match the expected package. Any chance someone could update the main README.md with the updated process to import the project into Eclipse?

Thanks so much!

**Comment by MatrixFrog** [comment]
You might just need to tell Eclipse to ignore that `super/` directory.

**Comment by hoyt85** [comment]
Yeah, I tried that, but then the maven build generates a bunch of GWT "no source found for x" errors for all of the com.google.gson imports in the src/com/google/debugging/sourcemap/SourceMapObjectParser.java file (also added during that commit).

**Comment by MatrixFrog** [comment]
For normal development on the compiler, you probably don't care about the gwt version, so maybe you can tell Eclipse to old build the main maven project instead of the entire thing including the linter/gwt version/etc. I'm not sure what that setting would be in Eclipse unfortunately.

**Comment by hoyt85** [comment]
That's an interesting idea, I'll look into it. Still, I really think someone should update the main README.md since, currently, running "mvn eclipse:eclipse -DdownloadSources=true" and importing to Eclipse on a fresh git clone of the closure-compiler renders maven unable to build the compiler.

Steps to reproduce:
1. git clone https://github.com/google/closure-compiler.git
2. cd closure-compiler
3. mvn eclipse:eclipse -DdownloadSources=true
4. In Eclipse: Navigate to File > Import > Maven > Existing Maven Projects and browse to closure-compiler:
- Import both closure-compiler and the nested externs project.
- Disregard the warnings about maven-antrun-plugin and build errors.
5. Back in shell under closure-compiler directory: mvn

Results in the following error:
[ERROR] Manifest com.google.javascript:closure-compiler-linter:jar:1.0-SNAPSHOT : The default package '.' is not permitted by the Import-Package syntax. This can be caused by compile errors in Eclipse because Eclipse creates valid class files regardless of compile errors.

**Comment by ChadKillingsworth** [comment]
Most of us have moved away from Eclipse. I'd actually have to go download it just to try that out. We will happily take a PR though to correct the instructions.

**Comment by MatrixFrog** [comment]
Chad what are you using nowadays? IntelliJ?

**Comment by ChadKillingsworth** [comment]
I am - never looked back

**Comment by ChadKillingsworth** [comment]
It's not perfect though either. Really hates multiple maven files in the same folder.

**Comment by hoyt85** [comment]
I'm happy to switch to IDEA if that's what people are using. I'm only on eclipse because I noticed you guys provided instructions for an eclipse import so I assumed that's what people were using. I just think it's a good idea to provide instructions on how to import and build the project into one of the standard IDEs. Thanks for the responses guys, I really appreciate it!
